controllers/conditions.go

```diff
@@ -123,6 +129,25 @@ func (r *ImageClusterInstallReconciler) setHostConfiguredCondition(ctx context.C
 	return r.Status().Patch(ctx, ici, patch)
 }
 
+func (r *ImageClusterInstallReconciler) setHostValidationCondition(ctx context.Context, ici *v1alpha1.ImageClusterInstall, err error) error {
+	cond := hivev1.ClusterInstallCondition{
+		Type:    hivev1.ClusterInstallConditionType(v1alpha1.HostValidatedCondition),
+		Status:  corev1.ConditionTrue,
+		Reason:  v1alpha1.HostValidationSucceeded,
+		Message: v1alpha1.HostValidationsOKMsg,
+	}
```

Comment:
What does installation condition have to do with it?

Reply:
Same as with image ready one, this is type we use